ql/src/java/org/apache/hadoop/hive/ql/exec/repl/incremental/IncrementalLoadTasksBuilder.java

```diff
@@ -70,9 +70,10 @@
   private final HiveConf conf;
   private final ReplLogger replLogger;
   private static long numIteration;
+  private Long eventTo;
```

Comment:
Can we change it to final type? same as other members in this class.

Reply:
done